### ISSUE TITLE
can't use back button on login page

### ISSUE THREAD

**Issue by twobraids** [mozilla-iot/gateway#1072]
Go to the gateway.local login page using an existing tab with some browsing history.  Try to use the back button.  The result is always that the login page reloads.

**Comment by benfrancis** [comment]
I assume this is because gateway.local redirects to gateway.local/login if you're not logged in. Clicking the back button will go back to gateway.local which will redirect to gateway.local/login.

Pressing the back button twice quickly will go back to the page you visited before gateway.local

Not really sure whether to consider this a bug.

**Comment by twobraids** [comment]
Breaking the back button is a UX bug.  A fast well timed double click is not a solution for those of us with arthritis.  Most Web sites that require a login have implementations that avoid the problem.  A click-to-login link or integrating the login into the home page could be a solution.

As such, this is not a particularly high priority problem.

**Comment by benfrancis** [comment]
I agree it's not ideal. One solution is to have the login page appear at the root URL rather than at /login, but that may break some things in the current implementation.

Also note that in Firefox you can also right click on the back button to go back further in history.
